### DISCUSSION TITLE
Compliance:  GitHub server locations (USA?) and data encryption

### DISCUSSION THREAD

**Discussion by rdalisay** [community/community#63197] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Before we can sign up for GitHub accounts, my company is requiring official documentation from GitHub to ensure that:

1. All of our data (including source code) will be stored only on servers located in the United States.
2. All data will be encrypted in transit and at rest.

I've been unable to find any documentation from GitHub about this.  GitHub Sales pointed me to GitHub Support.  However, the "GitHub Support" URL they shared is only to the self-serve knowledge base.  It seems that to get a human to answer my questions, I will have to sign up for a paid plan first.

Could someone please point me to GitHub official docs providing the above info?  Or, if you're from GitHub, answer them in an official capacity?  Thank you.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there, 

Thank you for your patience. I was reviewing our backlog of unanswered posts and I can share the following, all linked from our various documentation: 

1. [From our documentation](https://docs.github.com/en/enterprise-cloud@latest/admin/data-residency/about-github-enterprise-cloud-with-data-residency): 
> By default, GitHub stores data for GitHub.com in the USA. 

2. From the [GitHub Enterprise Trust Center](https://ghec.github.trust.page/): 
> All data including Git data, issues, comments, projects, attachments, pages, etc. is encrypted at rest with AES-256 encryption with keys managed by the Azure platform. <sup>[1](https://ghec.github.trust.page/faq?s=o8zx3hfxc6amct74xqv0)</sup>

> GitHub encrypts data in transit using TLSv1.2 or higher, and private data is transmitted exclusively via SSH or HTTPS (with SSLv* disabled). <sup>[2](https://ghec.github.trust.page/faq?s=o0jh3gio5vjxm7h23dpbac)</sup>

**Comment by BagasAjiSaputra** [comment]
insightful

**Comment by BagasAjiSaputra** [comment]
Insightfuls
